### PR TITLE
procedures: more options for minikube

### PR DESCRIPTION
**Pull request by themr0c**
## What does this pull request change?

Specify more options for the Minikube installer: more memory, more disk, enable dashboard.

## What issues does this pull request fix or reference?

Che installing on Minikube but behaving unreliably due to lack of resources.

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/main/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: modules/administration-guide/pages/installing-che-on-minikube.adoc [hunk 1]
```diff
@@ -21,10 +21,10 @@ WARNING: Remember that single-node {kubernetes} clusters are suited only for tes
 
 .Procedure
 
-. Start Minikube. Allocate at least 8GB of RAM and 4 CPU cores:
+. Start Minikube. Allocate at least 10GB of RAM and 4 CPU cores:
```
Comment (themr0c) [inline modules/administration-guide/pages/installing-che-on-minikube.adoc]:
```suggestion
. Start Minikube with required addons and sufficient resources:
```

File: modules/administration-guide/pages/installing-che-on-minikube.adoc [hunk 2]
```diff
@@ -21,10 +21,10 @@ WARNING: Remember that single-node {kubernetes} clusters are suited only for tes
 
 .Procedure
 
-. Start Minikube. Allocate at least 8GB of RAM and 4 CPU cores:
+. Start Minikube with required addons and sufficient resources:
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-minikube.adoc]:
```suggestion
. Start Minikube with required add-ons and sufficient resources:
```

**Review by deerskindoll — COMMENTED** [review]
one suggestion. check it out.

**Review by deerskindoll — APPROVED** [review]
lgtm

**Review by tolusha — APPROVED** [review]

**Pull request merged**